packages/foreman/rubygem-actioncable/rubygem-actioncable.spec

```diff
@@ -0,0 +1,94 @@
+# Generated from actioncable-5.2.0.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name actioncable
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 5.2.1
+Release: 1%{?dist}
+Summary: WebSocket framework for Rails
+Group:   Development/Languages
+License: MIT
+URL:     http://rubyonrails.org
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Obsoletes: tfm-ror52-rubygem-actioncable
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(actionpack)
+Requires: %{?scl_prefix}rubygem(nio4r)
+Requires: %{?scl_prefix}rubygem(websocket-driver)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby >= 2.2.2
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%{?scl:Requires: %{?scl_prefix}runtime}
+
+%description
+Structure many real-time application concerns into channels over a single
+WebSocket connection.
+
+
+%package doc
+Summary:   Documentation for %{name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+
+%files
+%dir %{gem_instdir}
+%license %{gem_instdir}/MIT-LICENSE
+%{gem_libdir}
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/CHANGELOG.md
+%doc %{gem_instdir}/README.md
+
+%changelog
+* Wed Aug 22 2018 Eric D. Helms <ericdhelms@gmail.com> 5.2.1-1
```

Comment:
I'd vote that we bump the release on each package so we have a changelog entry indicating the switch to tfm-

Reply:
Will do!